code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -433,6 +433,7 @@
 		CHECK_TICK
 
 /obj/item/storage/New()
+	..()

```

Comment:
I can't believe this hasn't been caught until now.

I also wonder how many things aren't functioning properly, because of this.

Reply:
IDEK what the implications could be. For curiosity's sake, what did/could this being missing end up causing?

Reply:
Well, the most obvious one was storage items couldn't initialize their actions (if they had any....none currently do).

Also means they're not: properly having armor set, having their object integrity set, having hitsounds set (if they have none), they lack an areaMaster, and they're not going to attempt initialization either, which could cause mapping issues.

Reply:
Thanks, I appreciate you clearing that up. Makes a lot more sense
I too now wonder how the hell stuff worked despite this having been an issue :thinking: